docs/tutorials/fundamentals/getting_started.md

```diff
@@ -57,24 +56,35 @@ but we'll be adding support for this functionality soon as discussed in [#379](h
 ### Python script usage
 
 To launch napari from a python script, inside your script you should import `napari`,
-and then create the `Viewer` by adding some data.
+then create a {class}`Viewer<napari.Viewer>` and {class}`Image<napari.layers.Image>`
+layer by adding some image data.
 
 For example, to add an image and some points inside your script you should include:
 
 ```python
 import napari
 
-# create a Viewer and add an image here
-viewer = napari.view_image(my_image_data)
+# create a `Viewer` and `Image` here
+viewer, image_layer = napari.imshow(my_image_data)
 
-# custom code to add data here
+# add points layer here
 viewer.add_points(my_points_data)
 
 # start the event loop and show the viewer
 napari.run()
 ```
 
-then run your script from the command line to launch the viewer with your data:
+Note that {func}`imshow<napari.imshow>` is a convenience function that is
+equivalent to:
+
+```python
+import napari
+
+viewer = napari.Viewer()
+image_layer = viewer.add_image(image)
+```
+
+You can now run your script from the command line to launch the viewer with your data:
```

Comment:
Nope, it won't run because `my_image_data` isn't defined. See comment above.
I think making it runnable is good!